packages/nodes-base/nodes/Form/utils.ts

```diff
@@ -216,6 +220,19 @@ const validateResponseModeConfiguration = (context: IWebhookFunctions) => {
 			},
 		);
 	}
+
+	if (isRespondToWebhookConnected) {
```

Comment:
Shouldn't this have a version check? If responseMode == 'responseNode' and version is <= 2.1 it would always throw now, no?

Reply:
indeed, thanks :pray: